### PR TITLE
fix(app, components) fix csv name wrapping issue

### PR DESCRIPTION
**Pull request by koji**
# Overview
fix csv name wrapping issue

close RQA-3516

Screenshot 2024-11-12 at 7 10 36 PM

Screenshot 2024-11-12 at 7 14 56 PM

## Risk assessment
low

### CODE REVIEW

File: app/src/organisms/ODD/ProtocolSetup/ProtocolSetupStep/index.tsx [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
   TEXT_ALIGN_RIGHT,
-  COLORS,
+  truncateString,
   TYPOGRAPHY,
-  NO_WRAP,
-  LegacyStyledText,
 } from '@opentrons/components'
 import { useToaster } from '../../../ToasterOven'
 
+const CSV_FILE_MAX_LENGTH = 15
```
Comment (ncdiehl11) [inline app/src/organisms/ODD/ProtocolSetup/ProtocolSetupStep/index.tsx]: Is this arbitrary or recommended from design?

**Review by ncdiehl11 — APPROVED** [review]
question regarding max length but looks good

File: app/src/organisms/ODD/ProtocolSetup/ProtocolSetupStep/index.tsx (hunk 1 above)
Comment (koji) [reply app/src/organisms/ODD/ProtocolSetup/ProtocolSetupStep/index.tsx]: I just counted the characters in the design.

https://www.figma.com/design/MlZjOO8aZXsw3HfTBoyw9U/Release%3A-CSV?node-id=2088-6307&m=dev

Comment (koji) [reply app/src/organisms/ODD/ProtocolSetup/ProtocolSetupStep/index.tsx]: oh, the max should be 18.

**Pull request merged**